app/admin/formwidgets/scheduleeditor/assets/js/scheduleeditor.js

```diff
@@ -28,6 +28,116 @@
+                            {name:"11",title:"11:00-11:15"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"11:15-11:30"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"11:30-11:45"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"11:45-12:00"},
+                            {name:"12",title:"12:00-12:15"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"12:15-12:30"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"12:30-12:45"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"12:45-13:00"},
+                            {name:"13",title:"13:00-13:15"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"13:15-13:30"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"13:30-13:45"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"13:45-14:00"},
+                            {name:"14",title:"14:00-14:15"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"14:15-14:30"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"14:30-14:45"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"14:45-15:00"},
+                            {name:"15",title:"15:00-15:15"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"15:15-15:30"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"15:30-15:45"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"15:45-16:00"},
+                            {name:"16",title:"16:00-16:15"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"16:15-16:30"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"16:30-16:45"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"16:45-17:00"},
+                            {name:"17",title:"17:00-17:15"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"17:15-17:30"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"17:30-17:45"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"17:45-18:00"},
+                            {name:"18",title:"18:00-18:15"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"18:15-18:30"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"18:30-18:45"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"18:45-19:00"},
+                            {name:"19",title:"19:00-19:15"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"19:15-19:30"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"19:30-19:45"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"19:45-20:00"},
+                            {name:"20",title:"20:00-20:15"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"20:15-20:30"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"20:30-20:45"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"20:45-21:00"},
+                            {name:"21",title:"21:00-21:15"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"21:15-21:30"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"21:30-21:45"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"21:45-22:00"},
+                            {name:"22",title:"22:00-22:15"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"22:15-22:30"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"22:30-22:45"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"22:45-23:00"},
+                            {name:"23",title:"23:00-23:15"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"23:15-23:30"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"23:30-23:45"},{name:"&nbsp;&nbsp;&nbsp;&nbsp;",title:"23:45-00:00"},
```

Comment:
Can you generate these dynamically instead of hardcoding?

Reply:
Yeah I need to do a bit of refactoring now the functionality is in place

Will tidy it all up